src/components/metadata/statics/StaticMetaSimple.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TextareaAutosize from 'react-textarea-autosize';
+
+export class StaticMetaSimple extends Component {
+  render() {
+    const { fieldValue } = this.props;
+    return (
+      <div className="meta-value">
+        <TextareaAutosize
+          className="field value-field"
+          value={`${fieldValue}`}
```

Comment:
```suggestion
          value={fieldValue}
```

Reply:
This is intentional.. The same also exists in the *non-static* version `MetaSimple`:
https://github.com/jekyll/jekyll-admin/blob/cbc5ecc90b7aaf1aee76b517793afd434c99f901/src/components/metadata/MetaSimple.js#L51

I'm guessing that this was to ensure that even Booleans are stringified..

Reply:
Hmm.. You're right. Leave it then 👍